src/main/java/org/tikv/cdc/RegionCDCClient.java

```diff
@@ -178,7 +179,7 @@ public void onNext(final ChangeDataEvent event) {
   }
 
   private void submitEvent(final CDCEvent event) {
-    LOGGER.debug("submit event: {}", event);
+    LOGGER.atDebug().log("submit event: {}", event);
```

Comment:
The original code is simpler and the performance is the same.
http://www.slf4j.org/faq.html#logging_performance